docs/guide/backend-integration.md

```diff
@@ -129,14 +137,13 @@ If you need a custom integration, you can follow the steps in this guide to conf
    ```
 
    Specifically, a backend generating HTML should include the following tags given a manifest
-   file and an entry point:
-   - A `<link rel="stylesheet">` tag for each file in the entry point chunk's `css` list
-   - Recursively follow all chunks in the entry point's `imports` list and include a
-     `<link rel="stylesheet">` tag for each CSS file of each imported chunk.
-   - A tag for the `file` key of the entry point chunk (`<script type="module">` for JavaScript,
-     or `<link rel="stylesheet">` for CSS)
-   - Optionally, `<link rel="modulepreload">` tag for the `file` of each imported JavaScript
-     chunk, again recursively following the imports starting from the entry point chunk.
+   file and an entry point. **Note that following this order is recommended for optimal performance, though not strictly required:**
+   1. A `<link rel="stylesheet">` tag for each file in the entry point chunk's `css` list (if it exists)
+   2. Recursively follow all chunks in the entry point's `imports` list and include a
+      `<link rel="stylesheet">` tag for each CSS file of each imported chunk's `css` list (if it exists). For CSS entry points, they won't have a `css` property, so you can check if the property exists before processing.
+   3. A tag for the `file` key of the entry point chunk. This can be `<script type="module">` for JavaScript, `<link rel="stylesheet">` for CSS, or appropriate tags for other file types (entry points can include HTML files, assets, etc.)
```

Comment:
```suggestion
   3. A tag for the `file` key of the entry point chunk. This can be `<script type="module">` for JavaScript, `<link rel="stylesheet">` for CSS.
```